cves/kernel/CVE-2021-45402.yml

```diff
@@ -55,7 +55,14 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  This vulnerbility is within the file that checks the validity of BPF instructions. 
+  BPF's are sandbox programs that allow for users to add extended functionality to linux kernal
+  without changing the linkux source code. 
```

Comment:
Great description, you just have a simple misspelling of Linux in this sentence.

Reply:
^^ and a misspelling of kernel

Reply:
Fixed thank you!